crates/astria-cli/src/cli/rollup.rs

```diff
@@ -61,8 +63,10 @@ pub struct ConfigCreateArgs {
     /// Optional. Will be derived from the rollup name if not provided
     #[clap(long = "rollup.chain-id", env = "ROLLUP_CHAIN_ID", required = false)]
     pub chain_id: Option<String>,
-    #[clap(long = "rollup.network-id", env = "ROLLUP_NETWORK_ID")]
+    /// The Network ID for the EVM chain
+    #[clap(long = "rollup.network-id", env = "ROLLUP_NETWORK_ID", default_value_t = DEFAULT_NETWORK_ID)]
     pub network_id: u64,
```

Comment:
is this the network id or the chain id for the evm? or both?

Reply:
The `network_id` is the evm network ID which you might lookup on chainlist. The `chain_id` is what is used through the shared sequencer. We will derive a `chain_id` from the `name` if not provided, where the name is just something human readable used as an identifier.

Reply:
sorry i guess i meant whether this is the `chain_id` or `network_id` for the EVM chain since in eth-land those are two different things (althought often used similarly), network_id is used for the p2p network while chain_id is what's on chainlist. i'm kinda thinking we should change our `chain_id` to be `rollup_id` since it's a bit of a naming conflict

Reply:
anyways out of scope for this PR